client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs

```diff
@@ -68,24 +69,41 @@ public abstract class DbCleanTestBase {
 
 
         public static void seedFKData(DbConnection connection, DatabaseType type = DatabaseType.H2) {
-            SqlScriptRunner.ExecCommand(connection, "CREATE TABLE Foo(x int, primary key (x));");
-            SqlScriptRunner.ExecCommand(connection, "CREATE TABLE Bar(y int, primary key (y));");
+            if (type == DatabaseType.MS_SQL_SERVER){
+                SqlScriptRunner.ExecCommand(connection, "CREATE SCHEMA Foo AUTHORIZATION dbo;");
+            }
+            SqlScriptRunner.ExecCommand(connection, "CREATE TABLE "+getFooTable(type)+"(x int, primary key (x));");
+            SqlScriptRunner.ExecCommand(connection, "CREATE TABLE "+getBarTable(type)+"(y int, primary key (y));");
 
             switch (type) {
                 case DatabaseType.MS_SQL_SERVER:
                 case DatabaseType.POSTGRES:
                     SqlScriptRunner.ExecCommand(connection,
-                        "alter table Bar add constraint FK foreign key (y) references Foo;");
+                        "alter table "+getBarTable(type)+" add constraint FK foreign key (y) references "+getFooTable(type)+";");
                     break;
                 case DatabaseType.MYSQL:
-                    SqlScriptRunner.ExecCommand(connection, "alter table Bar add foreign key (y) references Foo(x);");
+                    SqlScriptRunner.ExecCommand(connection, "alter table "+getBarTable(type)+" add foreign key (y) references "+getFooTable(type)+"(x);");
                     break;
                 default:
                     throw new InvalidOperationException("NOT SUPPORT");
             }
 
-            SqlScriptRunner.ExecCommand(connection, "INSERT INTO Foo (x) VALUES (42)");
-            SqlScriptRunner.ExecCommand(connection, "INSERT INTO Bar (y) VALUES (42)");
+            SqlScriptRunner.ExecCommand(connection, "INSERT INTO "+getFooTable(type)+" (x) VALUES (42)");
+            SqlScriptRunner.ExecCommand(connection, "INSERT INTO "+getBarTable(type)+" (y) VALUES (42)");
+        }
+
+        private static string getFooTable(DatabaseType type){
+            switch (type){
+                case  DatabaseType.MS_SQL_SERVER: return "Foo.Foo";
```

Comment:
add bit explanation on this

Reply:
even if is a test, it is confusing at this point. add a comment like "The tests using MS_SQL_SERVER are defining a schema, called Foo"

Reply:
this is added in a test `DbCleanerWithSchemaMsSqlServerTest`. please check methods `GetFooTable` and `GetBarTable` in the test